dashboard/app/models/user.rb

```diff
@@ -575,6 +575,12 @@ def reset_secret_words
     save!
   end
 
+  def all_advertised_scripts_completed?(advertised_scripts)
+    advertised_scripts.all? do |script|
```

Comment:
Style/SymbolProc: Pass `&:completed?` as an argument to `all?` instead of a block.